tests/src/test/scala/scalaz/ISetTest.scala

```diff
@@ -21,7 +21,8 @@ object ISetTest extends SpecLite {
   checkAll(band.laws[ISet[Int]])
```

Comment:
We can remove `checkAll(band.laws[ISet[Int]])` if `trait Semilattice extends Band` and `semilattice laws` include `band laws`